app/controllers/api/v1x0/requests_controller.rb

```diff
@@ -39,6 +39,8 @@ def request_params
       end
 
       def rbac_scope(relation)
+        index_access_check
+
```

Comment:
Above `RBAC::Access.enabled? ? collection(rbac_scope(reqs)) : collection(reqs)` can simplify to
`collection(rbac_scope(reqs))`, as you did in `actions_controller.rb`